cinder_volume/configuration.py

```diff
@@ -18,9 +18,23 @@
 takes as input from `snap set`.
 """
 
+from __future__ import annotations
```

Comment:
How is that needed?

The charm is running in python3.12 or later. I don't think we need that ?